config/environments/follower.rb

```diff
@@ -7,7 +7,7 @@
 require 'rack/remember_uuid'
 
 Rails.application.configure do
-  config.log_level = :info
+  config.log_level = ENV['CONJUR_LOG_LEVEL'] || :info
```

Comment:
why do we need this line if it's configured already in `production.rb`? don't you specify here only fields you're overriding or adding?